### PR TITLE
Add oclc icon

### LINKED ISSUE #12183: Request: OCLC
### Brand Name

OCLC

### Website

https://www.oclc.org

### Popularity Metric

Similarweb Global Rank: [3,737](https://www.similarweb.com/website/oclc.org)

### Official Resources for Icon and Color

- SVG: https://www.oclc.org/content/dam/oclc/design-images/navigation-logo.svg

### Additional Comments

Media kit available: https://www.oclc.org/en/news/media-kit.html (only EPS; same as #11514)

### PR DESCRIPTION
**Pull request by timmy471**
simpleicons (2)

**Issue:** closes #12183 

**Popularity metric:** 
Similarweb Global Rank: [3,737](https://www.similarweb.com/website/oclc.org)

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -18192,4 +18198,4 @@
             "source": "https://www.zyte.com"
         }
     ]
-}
+}
```
Comment (mondeja) [inline _data/simple-icons.json:18201]:
```suggestion
}

```

File: icons/oclc.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>OCLC</title><path d="M20.07 7.11C18.726 2.388 13.777-.688 8.55.133 2.973 1.008-.788 5.97.14 11.214c.635 3.679 3.407 6.52 6.918 7.6.061 1.68.958 3.307 2.565 4.303 2.612 1.629 6.147.94 7.855-1.543a5 5 0 0 0 .408-.687c2.727-.614 5.038-2.576 5.834-5.316.962-3.346-.632-6.785-3.651-8.46M7.115 17.852c-2.386-1.06-4.205-3.227-4.674-5.91-.751-4.312 2.206-8.36 6.629-9.05 3.153-.488 6.197.862 7.943 3.244-3.844-.436-7.556 1.823-8.621 5.455a7 7 0 0 0-.079 3.687 5 5 0 0 0-.323.42 5.14 5.14 0 0 0-.874 2.151zm2.229-3.514a5.8 5.8 0 0 1 .25-1.66c.973-3.322 4.513-5.314 7.922-4.422 3.41.892 5.4 4.294 4.422 7.632-.56 1.919-1.964 3.38-3.71 4.111.605-2.162-.286-4.541-2.385-5.841-2.043-1.274-4.639-1.13-6.499.18m6.866 6.96c-1.357 1.954-4.076 2.531-6.078 1.297s-2.528-3.835-1.165-5.79c1.357-1.954 4.076-2.531 6.079-1.297 2.002 1.25 2.522 3.85 1.164 5.79" /></svg>
```
Comment (mondeja) [inline icons/oclc.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>OCLC</title><path d="M20.07 7.11C18.726 2.388 13.777-.688 8.55.133 2.973 1.008-.788 5.97.14 11.214c.635 3.679 3.407 6.52 6.918 7.6.061 1.68.958 3.307 2.565 4.303 2.612 1.629 6.147.94 7.855-1.543a5 5 0 0 0 .408-.687c2.727-.614 5.038-2.576 5.834-5.316.962-3.346-.632-6.785-3.651-8.46M7.115 17.852c-2.386-1.06-4.205-3.227-4.674-5.91-.751-4.312 2.206-8.36 6.629-9.05 3.153-.488 6.197.862 7.943 3.244-3.844-.436-7.556 1.823-8.621 5.455a7 7 0 0 0-.079 3.687 5 5 0 0 0-.323.42 5.14 5.14 0 0 0-.874 2.151zm2.229-3.514a5.8 5.8 0 0 1 .25-1.66c.973-3.322 4.513-5.314 7.922-4.422 3.41.892 5.4 4.294 4.422 7.632-.56 1.919-1.964 3.38-3.71 4.111.605-2.162-.286-4.541-2.385-5.841-2.043-1.274-4.639-1.13-6.499.18m6.866 6.96c-1.357 1.954-4.076 2.531-6.078 1.297s-2.528-3.835-1.165-5.79c1.357-1.954 4.076-2.531 6.079-1.297 2.002 1.25 2.522 3.85 1.164 5.79"/></svg>
```

**Review by mondeja — CHANGES_REQUESTED** [review]
You need to maintain the aspect ratio. Your icon's size is 24x24 and the source is 24x22,666. Please, block your tool when resizing.

image

**Comment by timmy471** [comment]
@mondeja could you please review the updated icon

**Pull request closed without merging**